tests/console/sblim_sfcb.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test if sfcbd has been migrated properly during the
+#          upgrade and still starts
+# Maintainer: Adam Majer <amajer@suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    assert_script_run 'rpm -qi sblim-sfcb';
+    assert_script_run 'systemctl start sblim-sfcb.service';
+    # FIXME: check if this service actually works, but this will do for now
+    sleep 10;
+    if (script_run('systemctl status sblim-sfcb.service | grep "Failed to load"') == 0) {
+        record_soft_failure('Migration went wrong and services were not unregistered - bnc#1041885');
```

Comment:
1. record_soft_failure is designed to workaround known bugs , from your comment I understand that it is not what you are looking for. If you want to add a check here do it with assert_script_run
2. script_run in line 23 will fail anyway because we autocomplete all bash calls with "-o pipefail"  so 'grep "Failed to load"' is dead code which will be never reached
3. Again because 'systemctl status <service_name>' return non-zero code when service is dead your assert_script_run will fail just after record_soft_failure so there is no point in record_soft_failure